src/components/charts/SwipeChart.tsx

```diff
@@ -201,7 +205,7 @@ const SwipeChartComponent = (params: Props) => {
           } else {
             const unixNow = Math.trunc(new Date().getTime() / 1000)
             const fromParam = unixNow - queryFromTimeOffset
-            const fetchPath = sprintf(DATASET_URL_4S, assetId, defaultFiat, fromParam, unixNow)
+            const fetchPath = sprintf(DATASET_URL_4S, assetId, 'USD', fromParam, unixNow)
```

Comment:
This will yield an inaccurate chart as all the chart points will be in USD. Even though you convert them live to the current fiat, their relative x/y coordinates will be off relative to the actual fiat price (usually a small amount, but still wrong). 

I would sooner query Gecko for the fiat if supported, and if not fall back to USD.

Reply:
This issue extends to our prior rates server changes. Historical-related prices are being scaled with current fiat-fiat prices

Hard-coding supported fiats for now, and making a task to fully fix the rates server in the same manner.